### PR TITLE
[Merged by Bors] - chore: remove unnecessary `Mathlib.Meta.Positivity` namespacing

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by YaelDillies — APPROVED** [review]
Thanks!

maintainer merge

File: Mathlib/Combinatorics/SimpleGraph/Regularity/Bound.lean [hunk 1]
```diff
@@ -267,24 +267,24 @@ theorem add_div_le_sum_sq_div_card (hst : s ⊆ t) (f : ι → 𝕜) (d : 𝕜)
 
 end SzemerediRegularity
 
-namespace Tactic
+namespace Mathlib.Meta.Positivity
```
Comment (eric-wieser) [inline Mathlib/Combinatorics/SimpleGraph/Regularity/Bound.lean:270]: @YaelDillies, I assume this change is fine?

**Comment by eric-wieser** [comment]
maintainer merge

File: Mathlib/Combinatorics/SimpleGraph/Regularity/Bound.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Combinatorics/SimpleGraph/Regularity/Bound.lean:270]: Hmm, yeah. This is a misport, since the convention changed in Lean 3.

**Pull request closed without merging**